Comparison Engine/Comparison Engine/Data/Data.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Comparison_Engine
 {
-    class Data
+    public class Data
```

Comment:
why was it not public before and if it wasn't public and you made this public, we might need to review all classes to see if they have privacy modifiers declared. Or was there a specific reason why you did this?

Reply:
> why was it not public before and if it wasn't public and you made this public, we might need to review all classes to see if they have privacy modifiers declared. Or was there a specific reason why you did this?

This change was made for style. We can revert this change if it doesn't match the style of the rest of the code.